package/carbonio-user-management.hcl

```diff
@@ -13,11 +13,11 @@ services {
     sidecar_service {
       proxy {
         local_service_address = "127.78.0.5"
-        upstreams = [
+        upstreams             = [
           {
-            destination_name = "carbonio-mailbox"
```

Comment:
Restore the upstream for carbonio mailbox, it's necessary for the service to work since it have to call soap endpoints from mailboxes.